scripts/setup_host.sh

```diff
@@ -252,6 +252,21 @@ function ubu_install_swtpm() {
     cd -
 }
 
+ubu_update_bt_fw(){
+    if [ -d "linux-firmware" ] ; then
+            rm -rf linux-firmware
+    fi
+    git clone https://kernel.googlesource.com/pub/scm/linux/kernel/git/firmware/linux-firmware
+    cd linux-firmware
+    git checkout 73144e02dcff54529ab42d951de682740d204180
```

Comment:
Could you add comments here to explain why need this special commit instead of latest? And maybe you can checkout a tag instead of a commit id?

Reply:
Could you add comments here to explain why need this special commit instead of latest? 
>> Comments added. 
 # Checkout to specific firmware version 22.50.0.4 as guest also uses this version. Latest
 # is not taken as firmware update process in the guest is manual

And maybe you can checkout a tag instead of a commit id?
>> Tag is not available for  22.50.0.2 firmware version